packages/core/src/Stores/client-store.js

```diff
@@ -2766,7 +2766,6 @@ export default class ClientStore extends BaseStore {
 
                 if (data?.passkeys_list) {
                     const should_show_effortless_login_modal =
-                        this.root_store.ui.is_mobile &&
```

Comment:
removed isMobile check from here and added to AppContent component  useeffect where we set setIsPasskeySupported